frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/PipelineBuilderForm.tsx

```diff
@@ -61,8 +64,16 @@ const PipelineBuilderForm: React.FC<PipelineBuilderFormProps> = (props) => {
   useExplicitPipelineTaskTouch();
 
   const statusRef = React.useRef(status);
+  const [container, setContainerView] = React.useState<HTMLElement>(null);
```

Comment:
Pretty sure this is the wrong hook. Refs are not variables, they are functions that return variables... this is the wrong hook I think.

```suggestion
  const [container, setContainerView] = React.useRef<HTMLElement>(null);
```